examples/pages/react/Menu.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+
+import * as Markup from '../../js/react';
+
+import {Button, Menu} from '../../../app-typescript';
+import {IMenuItem} from '../../../app-typescript/components/Menu';
+
+const menuItems: Array<IMenuItem> = [
+    {
+        label: 'Edit',
+        icon: 'icon-pencil',
+        onClick: () => console.log('action: Edit'),
+    },
+    {
+        label: 'Edit in the new window',
+        icon: 'icon-pencil',
+        onClick: () => console.log('action: Edit in the new window'),
+    },
+    {
+        label: 'Translate',
+        icon: 'icon-globe',
+        children: [
+            {
+                label: 'German',
+                onClick: () => console.log('action: German'),
+            },
+            {
+                label: 'French',
+                onClick: () => console.log('action: French'),
+            },
+        ],
+    },
+    {
+        label: 'Export',
+        icon: 'icon-download',
+        onClick: () => console.log('action: Export'),
+    },
+    {
+        label: 'Spike item',
+        icon: 'icon-trash',
+        onClick: () => console.log('action: Spike item'),
+    },
+    {separator: true},
+    {
+        label: 'Send to',
+        icon: 'icon-share-alt',
+        onClick: () => console.log('action: Send to'),
+    },
+    {
+        label: 'Export',
+        icon: 'icon-download',
+        onClick: () => console.log('action: Export'),
+    },
+    {
+        label: 'Spike item',
+        icon: 'icon-trash',
+        onClick: () => console.log('action: Spike item'),
+    },
+];
+
+export class MenuDocs extends React.Component {
+    render() {
+        return (
+            <section className='docs-page__container'>
```

Comment:
Can you add minimal example code at top like on other pages?

![image](https://user-images.githubusercontent.com/6507688/121552322-ac10c600-ca10-11eb-851e-b6632d71d383.png)

Also include small description and props list?

Reply:
I have added example code in the markup section.

Regarding props, there is an interface defined that specifies all possible configurations. Why rewrite the same info in the docs? Whoever is going to use a component can jump to the interface in a single click. I'm happy to add comments to interface properties if you find some unclear.

![go-to-interface](https://user-images.githubusercontent.com/2076953/121680438-db2f4200-cab9-11eb-8382-dba2d38805e5.gif)

Reply:
I agree, but we also use the props list so developers can easily see if component covers everything they need. All major frameworks use this approach so we want to have it also. All over components have it.

Reply:
Doesn't jumping to the interface allow to easily see the same info? :smile:

Regarding other frameworks, we can borrow good practices that are useful for us, but we don't need to copy everything. For example, a lot of frameworks are written in js and don't have interfaces, so documenting props makes sense for them. We do have interfaces, so we aren't getting much value from describing the same info in a different way. It's more of a maintenance burden for us.

@petrjasek what do you think?

Reply:
In that case you need to have imported that component to see if there is functionality you need. If you have everything listed, you can just go through components and see which one suits you the best. This is how we work, so please let's keep it consistant.

Reply:
Come on @tomaskikutis, please add the damn list. I don't see what the big deal is, it won't take much of your time. It'll probably take less effort than going into a discussion about it. I get that you or other developers might not need it but this is not just for developers. Non-developers, I included, can use the components to create more complex layouts, playground pages etc. It's good to have an overview without having to dig into the code. Please, let's keep the practice we started.